expirationd-scm-1.rockspec

```diff
@@ -17,7 +17,8 @@ dependencies = {
 build = {
     type = "builtin",
     modules = {
-        ["expirationd"] = "expirationd.lua",
+        ["expirationd"] = "expirationd/init.lua",
```

Comment:
Btw, I think it's okay to have `module.lua` in root and `module/` folder with subpackages. I'm sure we have a couple of such modules. Just never try to mix `module.lua` and `module/init.lua` and everything would be alright.

Reply:
I think it's up to you and @oleg-jukovec to decide which structure do you wish to use

Reply:
@DifferentialOrange , how is it usually done in other modules?
`module/init.lua` or `module/module.lua`?

Reply:
I would prefer to keep the code in one module folder.